relay/sources/rudder-package/tests/webapp_xml/disable_test1.xml.expected

```diff
@@ -10,4 +10,5 @@
     <Arg>org.eclipse.jetty.servlet.SessionPath</Arg>
     <Arg>/</Arg>
   </Call>
-</Configure>
\ No newline at end of file
+  <Set name="extraClasspath"></Set>
```

Comment:
Why is it left empty? Should not we remove it ?

Reply:
because it's easier to implement with a streaming parser :) I don't think it changes anything for the webapp.